### PR TITLE
Integrate Angular i18n finder

### PR DESCRIPTION
**Pull request by alicoding**
This patch add grunt angular i18n finder and making sure most of it using the right format
This way we will be able to run the grunt validate and find any missing or unused keys

To test this simply run `npm install` then run `grunt validate`

Please note that there are many unused strings that next PR will cover the fix

### CODE REVIEW

**Comment by alicoding** [comment]
@secretrobotron or @gesa can one of you help review this? Thanks!

File: public/views/appmaker.html [hunk 1]
```diff
@@ -13,24 +13,24 @@
...
-        <a href="//apps.webmaker.org/{{lang}}/designer" class="btn btn-info btn-lg">{{'Start from scratch' | i18n}}</a>
+        <a href="#starter-apps" class="btn btn-success btn-lg">{{ 'Pick a template' | i18n }}</a>
+        <a href="//apps.webmaker.org/{{lang}}/designer" class="btn btn-info btn-lg">{{ 'Start from scratch' | i18n }}</a>
       </p>
     </div>
   </section>
   <section name="starter-apps" id="starter-apps">
     <div class="container">
-      <p class="lead text-center">{{'Don\'t know where to start?' | i18n}}</p>
+      <p class="lead text-center">{{ 'Don't know where to start?' | i18n }}</p>
```
Comment (secretrobotron) [inline public/views/appmaker.html]: This looks wrong. Don't need to escape the apostrophe?

Comment (alicoding) [inline public/views/appmaker.html]: It doesn't like it when we do escape in the locale file @secretrobotron

Comment (alicoding) [inline public/views/appmaker.html]: Oh snap! I need to fix this.

**Comment by secretrobotron** [comment]
```
Running "angular_i18n_finder:files" (angular_i18n_finder) task
>> Found unused key names in JSON provided.
>>  Please consider removing them or add to the ignoreKeys.
>>  list: [ 'Don\'t know where to start?' ]
```

**Comment by secretrobotron** [comment]
Also, this is a bit weird:
```
Searching 9 files for "know where to start" (regex, case sensitive)

/Users/secretrobotron/src/webmaker.org/locale/en_US/appmaker.json:
    8      "description": "A short explanation for what Appmaker is"
    9    },
   10:   "Dont know where to start?": {
   11:     "message": "Don't know where to start?",
   12:     "description": "Don't know where to start?"
   13    },
   14    "Mobile apps made easy": {
   ..
   36      "description": "Title for Appmaker landing page"
   37    },
   38:   "Don't know where to start?": {
   39      "message": "Looking for an easy way to get started? Remix one of these templates to make your very own app.",
   40      "description": "Catch the imagination of users at the top of the page."

4 matches in 1 file
```

**Comment by alicoding** [comment]
nice catch @secretrobotron ! this is fixed.

**Pull request merged**